starlette/middleware/base.py

```diff
@@ -130,10 +127,6 @@ async def wrap(func: typing.Callable[[], typing.Awaitable[T]]) -> T:
 
                 return message
 
-            async def close_recv_stream_on_response_sent() -> None:
-                await response_sent.wait()
-                recv_stream.close()
-
```

Comment:
Remove this as per @graingert comment on https://matrix.to/#/!JfFIjeKHlqEVmAsxYP:gitter.im/$S0X8-1wH1qscoq6FQjVCVDgBL4Qk-5BcTnaQfqBCkGI?via=gitter.im&via=matrix.org&via=matrix.freyachat.eu